jupyterhub/values.yaml

```diff
@@ -182,7 +182,7 @@ proxy:
     enabled: true
     minAvailable: 1
   https:
-    enabled: true
+    enabled: false
```

Comment:
I think this makes sense as a default, but is something that probably breaks peoples configurations which left it out given our past documentation did so.

Hmmm, should we perhaps mitigate this? We could for example make template rendering fail using Helm's `fail` function given a certain condition, or we could provide a warning in NOTES.txt. For example, that hosts is specified and `https.enabled` is false, that would be a typical indication of an issue. Another one would be to not set a default value for this at all in values.yaml, and if it isn't rendering to be true or false but instead to be a blank string we assume it to be true...

Hmm... I'm not sure. What do you think about this potential need to help users transition?

Reply:
I think it would be fine to make it a breaking change, where we notify users of in the release notes, saying they need to avtively set it to enabled if they had it implicitly set enabled before.

I think it would also be fine to let one z2jh release look for a dependency on the default truthiness and actively fail template rendering if we think it would crash some configurations.